test_data_loader.py

```
@@ -0,0 +1,15 @@
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)